Formula/presto.rb

```diff
@@ -0,0 +1,99 @@
+class Presto < Formula
+  desc "Distributed SQL query engine for big data"
+  homepage "https://prestodb.io"
+  url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-server/0.150/presto-server-0.150.tar.gz"
+  sha256 "a3249691c16dc49395fb0effe1d407501cf54caf0db8d8596eab575e1867b30b"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  resource "presto-cli" do
+    url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/0.150/presto-cli-0.150-executable.jar"
+    sha256 "cf7032e820e88cce76e2cc92710a5bf82c459588c2d628165d084438e18a35ff"
+  end
+
+  def install
+    libexec.install Dir["*"]
+
+    (libexec/"etc/node.properties").write <<-EOS.undent
+      node.environment=production
+      node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
+      node.data-dir=#{var}/presto/data
+    EOS
+
+    (libexec/"etc/jvm.config").write <<-EOS.undent
+      -server
+      -Xmx16G
+      -XX:+UseG1GC
+      -XX:G1HeapRegionSize=32M
+      -XX:+UseGCOverheadLimit
+      -XX:+ExplicitGCInvokesConcurrent
+      -XX:+HeapDumpOnOutOfMemoryError
+      -XX:OnOutOfMemoryError=kill -9 %p
+    EOS
+
+    (libexec/"etc/config.properties").write <<-EOS.undent
+      coordinator=true
+      node-scheduler.include-coordinator=true
+      http-server.http.port=8080
+      query.max-memory=5GB
+      query.max-memory-per-node=1GB
+      discovery-server.enabled=true
+      discovery.uri=http://localhost:8080
+    EOS
+
+    (libexec/"etc/log.properties").write <<-EOS.undent
```

Comment:
If this goes under ~80 characters, which I think it _just_ about does, don't need the `EOS.undent` element.
```ruby
(libexec/"etc/log.properties").write "com.facebook.presto=INFO"
```
etc. We try to use the blocks only for multiple lines & not stretching formulae out too widely.